src/vs/workbench/contrib/terminal/browser/media/shellIntegration.zsh

```diff
@@ -25,50 +25,50 @@ fi
 # Shell integration was disabled by the shell, exit without warning assuming either the shell has
 # explicitly disabled shell integration as it's incompatible or it implements the protocol.
 if [ -z "$VSCODE_SHELL_INTEGRATION" ]; then
-	return
+	builtin return
```

Comment:
I believe `return` would be a reserved keyword not a builtin so this should be safe (and may break doing this).

Reply:
kitty does this and it seems to work

Reply:
https://github.com/kovidgoyal/kitty/blob/master/shell-integration/zsh/kitty-integration#L26